spec/models/product_spec.rb

```diff
@@ -245,12 +222,9 @@ def rate_field
         @product = TestProduct.create!(facility: @facility, name: "Test Product", url_name: "test")
         @price_group = FactoryGirl.create(:price_group, facility: @facility)
         @price_group2 = FactoryGirl.create(:price_group, facility: @facility)
-        @user = FactoryGirl.create(:user)
-        FactoryGirl.create(:user_price_group_member, user: @user, price_group: @price_group)
-        @user.reload
-
-        @user_price_group_ids = @user.price_groups.map(&:id)
+        @user_price_group_ids = [@price_group]
```

Comment:
I don't think we changed the name of the variable, although it might make sense to do so. If we are thinking of doing that, I would vote for getting rid of the instance variable altogether and either use a local variable or just put it inline (`expect(@product).to be_can_purchase([@price_group])`. That would put the expectations a little closer to where they're used. I don't feel too strongly about it, as I'd like to get this wrapped up soon.